### ISSUE TITLE
Python 3.7 Windows wheels?

### ISSUE THREAD

**Issue by Siecje** [pydata/numexpr#317]
I see there are wheels for Python 3.6 but not Python 3.7.

https://pypi.org/project/numexpr/#files

**Comment by robbmcleod** [comment]
We are waiting on 3.7.1 as there was some breaking bugs in 3.7.0.

**Comment by Siecje** [comment]
I ended up installing the wheel from here. https://www.lfd.uci.edu/~gohlke/pythonlibs/#numexpr

Is this a bad idea?

**Comment by cgohlke** [comment]
> If you don't call the test functions it should be ok.

?

```
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Numexpr version:   2.6.8
NumPy version:     1.14.5
Python version:    3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)]
Platform:          win32-AMD64-10.0.17134
CPU vendor:        GenuineIntel
CPU model:         Intel(R) Xeon(R) CPU E5-1650 v4 @ 3.60GHz
CPU clock speed:   3592 MHz
VML available?     True
VML/MKL version:   Intel(R) Math Kernel Library Version 2019.0.0 Product Build 20180829 for Intel(R) 64 architecture applications
Number of threads used by default: 8 (out of 12 detected cores)
Maximum number of threads: 64
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
----------------------------------------------------------------------
Ran 5520 tests in 4.781s

OK
```

**Comment by Siecje** [comment]
It looks like there will be a release candidate tomorrow.

https://www.python.org/dev/peps/pep-0537/#maintenance-releases

I'm curious which bug is it that you are waiting to be fixed? https://docs.python.org/3/whatsnew/changelog.html#windows

**Comment by bashtage** [comment]
https://bugs.python.org/issue34108

**Comment by robbmcleod** [comment]
We will need to wait for Appveyor to support 3.7.1 as well, FYI.

**Comment by Siecje** [comment]
3.7.1 is not available on AppVeyor yet https://www.appveyor.com/docs/windows-images-software/#python

**Comment by robbmcleod** [comment]
I raised an issue with Appveyor:

https://github.com/appveyor/ci/issues/2735

They've put it into their next milestone.

**Comment by Siecje** [comment]
Python 3.7.1 is now listed here https://www.appveyor.com/docs/windows-images-software/#python

**Comment by robbmcleod** [comment]
Ok thanks. I'll look into pushing a new release hopefully sometime in the next couple of days.

**Comment by robbmcleod** [comment]
2.6.9 is released with wheels for Python 3.7 for Windows:

https://pypi.org/project/numexpr/

Closing.
